stdpopsim/catalog/PonAbe/genome_data.py

```diff
@@ -1,36 +1,31 @@
-# File autogenerated from Ensembl REST API.
-# WARNING: autogenerated file has been edited!
+# File autogenerated from Ensembl REST API. Do not edit.
 data = {
     "assembly_accession": "GCA_002880775.3",
-    "assembly_name": "ponAbe3",  # Also known as Susie_PABv2.
+    "assembly_name": "Susie_PABv2",
     "chromosomes": {
-        "1": {"length": 227913704, "synonyms": []},
-        # Capitalisation of 2A/2B changed from the previous assembly,
-        # so the old names are here as synonyms.
-        "2A": {"length": 109511694, "synonyms": ["2a", "chr2a"]},
-        "2B": {"length": 129937803, "synonyms": ["2b", "chr2b"]},
-        "3": {"length": 193656255, "synonyms": []},
-        "4": {"length": 189387572, "synonyms": []},
-        "5": {"length": 179185813, "synonyms": []},
-        "6": {"length": 169501136, "synonyms": []},
-        "7": {"length": 145408105, "synonyms": []},
-        "8": {"length": 144036388, "synonyms": []},
-        "9": {"length": 112206110, "synonyms": []},
-        "10": {"length": 132178492, "synonyms": []},
-        "11": {"length": 128122151, "synonyms": []},
-        "12": {"length": 132184051, "synonyms": []},
-        "13": {"length": 98475126, "synonyms": []},
-        "14": {"length": 88963417, "synonyms": []},
-        "15": {"length": 82547911, "synonyms": []},
-        "16": {"length": 68237989, "synonyms": []},
-        "17": {"length": 75914007, "synonyms": []},
-        "18": {"length": 75923960, "synonyms": []},
-        "19": {"length": 57575784, "synonyms": []},
-        "20": {"length": 60841859, "synonyms": []},
-        "21": {"length": 34683425, "synonyms": []},
-        "22": {"length": 35308119, "synonyms": []},
-        "X": {"length": 151242693, "synonyms": []},
-        # Mitochondria absent in ponAbe3, so length taken from ponAbe2.
-        "MT": {"length": 16499, "synonyms": []},
```

Comment:
looks like we need to stick in this manually again?

Reply:
nope - it's [there](http://rest.ensembl.org/info/assembly/pongo_abelii?synonyms=1), just not listed in the karyotype:
```
  - 
    coord_system: primary_assembly
    length: 16499
    name: MT
```